static/app/views/discover/results.tsx

```diff
@@ -680,11 +685,29 @@ export class Results extends Component<Props, State> {
                 {this.renderError(error)}
                 {this.renderTips()}
                 {this.renderForcedDatasetBanner()}
-                <StyledPageFilterBar condensed>
-                  <ProjectPageFilter />
-                  <EnvironmentPageFilter />
-                  <DatePageFilter />
-                </StyledPageFilterBar>
+
+                <Wrapper>
+                  <Feature
+                    organization={organization}
+                    features="performance-discover-dataset-selector"
+                  >
+                    {({hasFeature}) =>
+                      hasFeature && (
+                        <DatasetSelector
+                          isHomepage={isHomepage}
+                          savedQuery={savedQuery}
```

Comment:
I thought we needed to initially, but we don't to because we set homepageQuery as the saved query [here](https://github.com/getsentry/sentry/blob/master/static/app/views/discover/savedQuery/index.tsx#L282) so the state we need gets passed down with savedQuery